resources/common/common.tf

```diff
@@ -148,6 +148,7 @@ data "terraform_remote_state" "etcd" {
         bucket = "${var.remote_state_bucket}"
         key = "etcd.tfstate"
         region = "${var.remote_state_region}"
+        role_arn = "${var.aws_account["role_arn"]}"
```

Comment:
these `role_arn` entries in `common.tf` are a bit of a problem that I'm not sure how to solve yet.